frontend/src/components/common/Filter/common/constants.ts

```diff
@@ -1,4 +1,1121 @@
+              ontology_term_id: "MmusDv:0000112",
+            },
+          ],
+        },
+        {
+          label: "Mature (6+ weeks)",
+          ontology_term_id: "MmusDv:0000110",
+          children: [
+            {
+              label: "Early adulthood (6 weeks–7 months)",
+              ontology_term_id: "MmusDv:0000061",
+            },
+            {
+              label: "Late adulthood (7+ months)",
+              ontology_term_id: "MmusDv:0000097",
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  [ONTOLOGY_VIEW_KEY.UBERON]: [
+    {
+      label: "Embryo",
+      ontology_term_id: "UBERON:0000068",
+      children: [
+        {
+          label: "Zygote",
+          ontology_term_id: "UBERON:0000106",
+        },
+        {
+          label: "Cleavage",
+          ontology_term_id: "UBERON:0000107",
+        },
+        {
+          label: "Blastula",
+          ontology_term_id: "UBERON:0000108",
+        },
+        {
+          label: "Gastrula",
+          ontology_term_id: "UBERON:0000109",
+        },
+        {
+          label: "Neurula",
+          ontology_term_id: "UBERON:0000110",
+        },
+        {
+          label: "Organogenesis",
+          ontology_term_id: "UBERON:0000111",
+        },
+        {
+          label: "Late embryonic",
+          ontology_term_id: "UBERON:0007220",
+        },
+      ],
+    },
+    {
+      label: "Post embryonic",
+      ontology_term_id: "UBERON:0000092",
+      children: [
+        {
+          label: "Larval",
+          ontology_term_id: "UBERON:0000069",
+        },
+        {
+          label: "Pupal",
+          ontology_term_id: "UBERON:0000070",
+        },
+        {
+          label: "Nursing",
+          ontology_term_id: "UBERON:0018685",
+        },
+        {
+          label: "Fully formed",
+          ontology_term_id: "UBERON:0000066",
+          children: [
+            {
+              label: "Sexually immature",
+              ontology_term_id: "UBERON:0000112",
+            },
+            {
+              label: "Post-juvenile adult",
+              ontology_term_id: "UBERON:0000113",
+            },
+          ],
+        },
+      ],
+    },
+  ],
+};
+/* eslint-enable sort-keys -- disabling key order for readability. */
+
+/**
+ * List of ethnicity ontology labels to exclude from filter functionality.
+ */
+export const ETHNICITY_DENY_LIST = ["na"];
+
+/**
+ * String value to append to labels in multi-panel categories if the value appears in more than one panel.
+ */
+export const LABEL_SUFFIX_NON_SPECIFIC = ", non-specific";
+
+/**
+ * Possible set of values that publication dates can be binned into.
+ */
+export const PUBLICATION_DATE_VALUES: number[] = [1, 3, 6, 12, 24, 36];
+
+/* eslint-disable sort-keys -- disable key order, contents of this file are generated */
+/* eslint-disable sonarjs/no-duplicate-string -- disable no dupes, this value is generated */
+export const TISSUE_DESCENDANTS: OntologyDescendants = {
```

Comment:
`TISSUE_DESCENDANTS` facilitates cross-panel filtering in multi-panel category filters.